hw/ip/hmac/dv/hmac_sim_cfg.hjson

```diff
@@ -68,25 +68,25 @@
     {
       name: hmac_test_sha_vectors
       uvm_test_seq: hmac_test_vectors_sha_vseq
-      run_opts: ["+test_vectors_dir={proj_root}/hw/dv/sv/test_vectors"]
+      run_opts: ["+test_vectors_dir={proj_root}/hw/dv/sv/test_vectors/"]
```

Comment:
It is weird to place assumption that the dir path here needs to add a '/' at the end. Perhaps fix this in the test_vectors_pkg instead?

Reply:
Oh I see, sure I will change it. Thanks for the suggestion!